code/modules/food/food/thecake.dm

```diff
@@ -0,0 +1,298 @@
+// Chaos cake
+
+/datum/recipe/chaoscake_layerone
+	reagents = list("flour" = 300,"milk" = 200, "sugar" = 100, "egg" = 30)
+	fruit = list("poisonberries" = 15, "cherries" =  15)
+	items = list(
+			/obj/item/weapon/reagent_containers/food/snacks/meat/,
+			/obj/item/weapon/reagent_containers/food/snacks/meat/,
+			/obj/item/weapon/reagent_containers/food/snacks/meat/,
+			/obj/item/weapon/reagent_containers/food/snacks/meat/,
+			/obj/item/weapon/reagent_containers/food/snacks/carpmeat,
+			/obj/item/weapon/reagent_containers/food/snacks/carpmeat/,
+			/obj/item/weapon/reagent_containers/food/snacks/carpmeat/,
+			/obj/item/weapon/reagent_containers/food/snacks/carpmeat/
+		)
+	result = /obj/structure/chaoscake
+
+/datum/recipe/chaoscake_layertwo
+	reagents = list("flour" = 300, "milk" = 200, "sugar" = 100, "egg" = 30, )
+	fruit = list("vanilla" =  15, "banana" = 15)
+	items = list(
+			/obj/item/weapon/reagent_containers/food/snacks/dough,
+			/obj/item/weapon/reagent_containers/food/snacks/dough,
+			/obj/item/weapon/reagent_containers/food/snacks/dough,
+			/obj/item/weapon/reagent_containers/food/snacks/dough
+		)
+	result = /obj/item/weapon/chaoscake_layer
+
+/datum/recipe/chaoscake_layerthree
+	reagents = list("flour" = 240, "milk" = 150, "sugar" = 80, "egg" = 24, "deathbell" = 100)
+	fruit = list("grapes" = 30)
+	items = list(
+			/obj/item/weapon/reagent_containers/food/snacks/dough,
+			/obj/item/weapon/reagent_containers/food/snacks/dough,
+			/obj/item/weapon/reagent_containers/food/snacks/dough
+		)
+	result = /obj/item/weapon/chaoscake_layer/three
+
+/datum/recipe/chaoscake_layerfour
+	reagents = list("flour" = 240, "milk" = 150, "sugar" = 80, "egg" = 24, "milkshake" = 300)
+	fruit = list("rice" = 30)
+	items = list(
+			/obj/item/weapon/reagent_containers/food/snacks/dough,
+			/obj/item/weapon/reagent_containers/food/snacks/dough,
+			/obj/item/weapon/reagent_containers/food/snacks/dough
+		)
+	result = /obj/item/weapon/chaoscake_layer/four
+
+/datum/recipe/chaoscake_layerfive
+	reagents = list("flour" = 180, "milk" = 100, "sugar" = 60, "egg" = 18, "blood" = 300)
+	fruit = list("tomato" = 20)
+	items = list()
+		/*
+			/obj/item/weapon/reagent_containers/food/snacks/lobster, Remind me to port lobster later

```

Comment:
Amusing, Travis still seems to not like that this comment line doesn't have a comma explicitly be at the end of the line regardless of whether the first part of a line is code or not. Try adding a comma to appease Travis.